examples/public/misc/popups-video-lealfet.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
```

Comment:
Tried the branch in a dev center staging deploy, and it could not find this example because I guess there is a typo in the filename and it should be `popups-video-leaflet.html`